Parsers/ASimWebSession/Parsers/vimWebSessionCiscoMeraki.yaml

```diff
@@ -0,0 +1,249 @@
+Parser:
+  Title: Web Session ASIM filtering parser for Cisco Meraki
+  Version: '0.1'
+  LastUpdated: Jun 20 2023
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco Meraki Documentation
+  Link: https://documentation.meraki.com/
```

Comment:
can we add direct link to Websession specific logs?

Reply:
Done